### PR TITLE
Update stream_update_for_matching_engine.ipynb

### PR DESCRIPTION
**Pull request by peterping666**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Change "allow_list" to "allow" for index creation as allow_list is not supported but allow is supported for index creation.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [x] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [x] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb [hunk 1]
```diff
@@ -525,7 +525,7 @@
         "        f.write('{\"id\":\"' + str(i) + '\",')\n",
```
Comment (andrewferlitsch) [inline notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: <p>combine the three code cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: <p>Create online queries</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Done.

[the remark this answers is not in the document]
Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Done

[the remark this answers is not in the document]
Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Done

[the remark this answers is not in the document]
Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Yeah, that's what I mean. Updated to "Vertex AI Matching Engine".

[the remark this answers is not in the document]
Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Updated.

Comment (peterping666) [reply notebooks/community/matching_engine/stream_update_for_matching_engine.ipynb:525]: Which three code cells are you referring to here?

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**